src/components/Editable.js

```diff
@@ -76,7 +76,7 @@ const stopPropagation = e => e.stopPropagation()
  * An editable thought with throttled editing.
  * Use rank instead of headRank(thoughtsRanked) as it will be different for context view.
  */
-const Editable = ({ disabled, isEditing, thoughtsRanked, contextChain, cursorOffset, showContexts, rank, style, dispatch }) => {
+const Editable = ({ disabled, isEditing, thoughtsRanked, contextChain, cursorOffset, showContexts, rank, uuid, style, dispatch }) => {
```

Comment:
Can the uuid be derived from `thoughtsRanked`?

Reply:
Yes. `uuid` is just redundant here. I will remove this.